fda-model/paireddata/PairedData.cs

```diff
@@ -12,29 +12,36 @@ public class PairedData: IPairedData
         //or array.reverse
         //we need the right comparer to sort the right way the first way 
         public PairedData(double[] xs, double[] ys){
-            if (!IsMonotonicallyIncreasing(xs))
+            if (xs != null && ys != null)
```

Comment:
i would encourage considering the pattern in uncertain paired data for nullness

https://github.com/HydrologicEngineeringCenter/fda-model/blob/216c72d6019366b83da676ff9c6a930909bad1f5/fda-model/paireddata/UncertainPairedData.cs#L24

Reply:
It looks like IsNull is a property set by the user? It feels like I have a different objective here - validation - whereas IsNull in uncertain paired data feels more like a part of construction?

Reply:
well the user is not going to set any of this, the viewmodel should build an object that can be validated. i think we should probably talk to figure it out